src/Kovan-DssSpell.sol

```diff
@@ -36,138 +27,46 @@ contract SpellAction {
     //
     // The contracts in this list should correspond to MCD core contracts, verify
     //  against the current release list at:
-    //     https://changelog.makerdao.com/releases/kovan/1.1.4/contracts.json
+    //     https://changelog.makerdao.com/releases/kovan/active/contracts.json
 
     ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
-    address constant FLIP_FAB        = 0x7c890e1e492FDDA9096353D155eE1B26C1656a62;
+    address constant MCD_ADM            = 0x27E0c9567729Ea6e3241DE74B3dE499b7ddd3fe6;
+    address constant VOTE_PROXY_FACTORY = 0x1400798AA746457E467A1eb9b3F3f72C25314429;
 
-    address constant GUSD            = 0x31D8EdbF6F33ef858c80d68D06Ec83f33c2aA150;
-    address constant MCD_JOIN_GUSD_A = 0x0c6B26e6AB583D2e4528034037F74842ea988909;
-    address constant MCD_FLIP_GUSD_A = 0xf6c0e36a76F2B9F7Bd568155F3fDc53ff1be1Aeb;
-    address constant PIP_GUSD        = 0xb6630DE6Eda0f3f3d96Db4639914565d6b82CfEF;
+    function execute() external {
+        address MCD_PAUSE   = CHANGELOG.getAddress("MCD_PAUSE");
+        address FLIPPER_MOM = CHANGELOG.getAddress("FLIPPER_MOM");
+        address OSM_MOM     = CHANGELOG.getAddress("OSM_MOM");
```

Comment:
Since we removed the deployer EOA address from the changelog, I thought I would re-validate these changelog addresses for mainnet:

```
seth call 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F 'getAddress(bytes32)(address)' $(seth --to-bytes32 "$(seth --from-ascii "MCD_PAUSE")")
0xbE286431454714F511008713973d3B053A2d38f3
seth call 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F 'getAddress(bytes32)(address)' $(seth --to-bytes32 "$(seth --from-ascii "FLIPPER_MOM")")
0xc4bE7F74Ee3743bDEd8E0fA218ee5cf06397f472
seth call 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F 'getAddress(bytes32)(address)' $(seth --to-bytes32 "$(seth --from-ascii "OSM_MOM")")
0x76416A4d5190d071bfed309861527431304aA14f
```

All good.  💪